book/content/temperatures/temperatures_basic.md

```diff
@@ -0,0 +1,211 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.3
+kernelspec:
+  display_name: festim-workshop
+  language: python
+  name: python3
+---
+
+# Basic functionality #
+
+Definition of a temperature field or problem is essential for hydrogen transport and FESTIM as a whole. Users can define it as a fixed value or as a function of space and time. Read more about temperature in FESTIM _[here](https://festim.readthedocs.io/en/fenicsx/userguide/temperature.html)_. FESTIM's unit for temperature is Kelvin.
+
+This tutorial will go over the basics of defining the temperature in a FESTIM simulation.
+
+Objectives:
+* Learn how to define homogeneous temperature
+* Learn how to define space and time dependent temperature functions
+* Learn how to define temperature as a `dolfinx.fem.Function`
+
++++
+
+## Defining temperatures in FESTIM ##
+
+The simplest way to define temperature is as a homogeneous value:
+
+```{code-cell} ipython3
+import festim as F
+
+my_model = F.HydrogenTransportProblem()
+my_model.temperature = 300  # K 
+```
+
+To define a space-dependent temperature function we can use ` lambda ` functions:
+
+$$ 
+
+T(x) = T_0 e^{-x}
+
+$$
+
+```{code-cell} ipython3
+import festim as F
+import ufl
+
+T0 = 300
+my_model = F.HydrogenTransportProblem()
+my_model.temperature = lambda x: T0 * ufl.exp(-x)  # K
+```
+
+For a time-dependent function such as:
+
+$$
+
+T(t) = T_0 sin(t)
+
+$$
+
+```{code-cell} ipython3
+my_model.temperature = lambda t: T0 * ufl.sin(t)
+```
+
+For a function in $ x, y, z, t $ space:
+
+$$
+
+T(x,y,z,t) = T_0 cos(x)sin(t) + 5y - 2z
+
+$$
+
+```{code-cell} ipython3
+my_model.temperature = lambda x,t: T0 * ufl.cos(x[0])*ufl.sin(t) + 5*x[1] - 2*x[2]
+```
```

Comment:
I might be tempted to remove this 3rd example all together and integrate something similar into the example below